docs/docsite/rst/plugins/filter.rst

```diff
@@ -21,7 +21,29 @@ You can add a custom filter plugin by dropping it into a ``filter_plugins`` dire
 Using filter plugins
 --------------------
 
-For information on using filter plugins, see :ref:`playbooks_filters`.
+You can use filters anywhere you can use templating in Ansible: in a play, in variables file, or in a Jinja2 template for the :ref:`template <template_module>` module. For more information on using filter plugins, see :ref:`playbooks_filters`.  Filters can return any type of data, but if you want to always return a boolean (C(True) or C(False)) you should be looking at a test instead.
+
+.. code-block:: YAML+Jinja
+
+  vars:
+     yaml_string: "{{ some_variable|to_yaml}}"
+
+Filters are the prefered way to manipulate data in Ansible, you can identify a filter because it is normally preceded by a C(|), with the expression on the left of it being the first input of the filter. Additional parameters may be passed into the filter itself as you would to most programming functions. These parameters can be either C(positional) (passed in order) or C(named) (passed as key=value pairs). When passing both types, positional arguments should go first.
+
+.. code-block:: YAML+Jinja
+
+   passing_positional: {{ (x == 32) | ternary('x is 32', 'x is not 32') }}
+   passing_extra_named_parameters: {{ some_variable | to_yaml(indent=8, width=1337) }}
+   passing_both: {{ some_variable| ternary('true value', 'false value', none_val='NULL') }}
+
+In the documentation filters will always have an C(_input) option that corresponds to the expression to the left of c(|). A C(positional:) field in the documentation will show which options are positional and in which order they are required.
```

Comment:
```suggestion
In the documentation filters will always have an ``_input`` option that corresponds to the expression to the left of ``|``. A ``positional:`` field in the documentation will show which options are positional and in which order they are required.
```

Reply:
_input is an option name

Reply:
C() does not work in RST files. You need to use double backticks.

(Also please note that there are 22 more suggestions from my review that the GH UI hid.)